pkg/api/store/cluster/cluster_store.go

```diff
@@ -669,6 +673,44 @@ func enableLocalBackup(data map[string]interface{}) {
 	}
 }
 
+func handleScheduleScanScheduleConfig(data map[string]interface{}) {
+	scheduleConfigData, ok := values.GetValue(data, "scheduledClusterScan", "scheduleConfig")
+	if !ok || (ok && scheduleConfigData == nil) {
+		return
+	}
+	scheduleConfig := convert.ToMapInterface(scheduleConfigData)
+	if scheduleConfig["cronSchedule"] == "" {
+		values.PutValue(data, cis.DefaultCronSchedule, "scheduledClusterScan", "scheduleConfig", "cronSchedule")
+	}
+}
+
+func handleScheduleScanScanConfig(data map[string]interface{}) {
+	scanConfigData, ok := values.GetValue(data, "scheduledClusterScan", "scanConfig")
+	if !ok || (ok && scanConfigData == nil) {
+		return
+	}
+	cisScanConfigData, ok := values.GetValue(data, "scheduledClusterScan", "scanConfig", "cisScanConfig")
+	if !ok || (ok && cisScanConfigData == nil) {
+		return
+	}
+	cisScanConfig := convert.ToMapInterface(cisScanConfigData)
+	if cisScanConfig["profile"] == "" {
+		values.PutValue(data, v3.CisScanProfileTypePermissive, "scheduledClusterScan", "scanConfig", "cisScanConfig", "profile")
+	}
+}
+
+func handleScheduledScan(data map[string]interface{}) {
+	scheduledClusterScan, ok := values.GetValue(data, "scheduledClusterScan")
+	if ok && scheduledClusterScan != nil {
+		enabled := values.GetValueN(data, "scheduledClusterScan", "enabled")
+		if enabled == nil || (enabled != nil && !enabled.(bool)) {
```

Comment:
nit: you can drop `enabled != nil`, if it were nil this wouldn't be evaluated

Reply:
I did think about that, I felt having that additional condition made the code easier to read.